src/main/java/net/minecraftforge/common/loot/LootModifierManager.java

```diff
@@ -56,7 +53,7 @@
 
 public class LootModifierManager extends JsonReloadListener {
     public static final Logger LOGGER = LogManager.getLogger();
-    private static final Gson GSON_INSTANCE = (new GsonBuilder()).registerTypeHierarchyAdapter(ILootFunction.class, LootFunctionManager.func_237450_a_()).registerTypeHierarchyAdapter(ILootCondition.class, LootConditionManager.func_237474_a_()).create();
+    private static final Gson GSON_INSTANCE = LootSerializers.func_237387_b_().create();
```

Comment:
The `GsonBuilder` returned by `LootSerializers.func_237387_b_` does not have a hierarchy adapter for `ILootCondition`. Is this a problem?

Reply:
Yes, it does, notice how `func_237387_b_` first calls `func_237386_a_`

Reply:
You are right